apps/web/src/components/Bootcamp/HowItWorks/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { HowItWorksCard } from 'apps/web/src/components/Bootcamp/HowItWorks/HowItWorksCard';
+
+const featureItems = [
+  {
+    number: '1',
+    title: 'Apply',
+    description: (
+      <div className="text-md font-sans text-white">
+        <p>Are you a crypto-proficient software engineer interested in learning to build smart contracts? Start out by applying to Base Bootcamp.</p>
+      </div>
+    ),
+  },
+  {
+    number: '2',
+    title: 'Onboard',
+    description: 'Upon acceptance, you’ll be invited to the Base Bootcamp Onboarding Course where you will receive all relevant onboarding materials (student handbook, mentor-pairing info, invite to the private Base Bootcamp Discord channel, and invite to the launch day).',
+  },
+  {
+    number: '3',
+    title: 'Kickoff',
+    description: 'Gather virtually for our initial kick-off. We’ll pair you with your mentor (an experienced Smart Contract Developer) and you’ll start the program.',
+  },
+  {
+    number: '4',
+    title: 'Build!',
+    description: (
+      <div className="text-md font-sans text-white">
+        <p>Work through <a className="underline" target="_blank" href="https://docs.base.org/base-camp/docs/welcome">Base Camp</a>, meeting regularly with your mentor. You’ll build your final project during the final two weeks - a real-world dapp that you’ll demo virtually on Demo Day.
```

Comment:
Sorry, this ought to have a linter on it, I'll try to find and configure one. Anytime we use `target="blank"` for security reasons we also ought to include `rel="noreferrer noopener"`. Could U go through the links in this PR and make sure they all have that?

Reply:
For sure - thanks!

Reply:
Thanks so much, again sorry for the inconvenience, this ought to be a linter autofix in the future

Reply:
Weird - it looks like I (something) added it to some of them?